src/ProjectNeon/Assets/Features/Combatants/MemberState.cs

```diff
@@ -44,6 +46,8 @@ public MemberState(IStats baseStats)
     public void TakePhysicalDamage(float amount) => ChangeHp((-(amount * ((1f - CurrentStats.Armor()) / 1f))) * CurrentStats.Damagability());
     private void ChangeHp(float amount) => Counter(TemporalStatType.HP).ChangeBy(amount);
 
+    public void Stun() => this.statuses.Add(Status.Stunned);
```

Comment:
Per the glossary, Stun is supposed to have a numerical component `X`. This doesn't receive the numerical part.

Reply:
Also, this doesn't seem to expire or be utilized yet. How is the next action being prevented?